.scrutinizr.yml

```diff
@@ -0,0 +1,34 @@
+# language: php
+
+filter:
+    paths: [./*]
+    excluded_paths: [./node_modules/*, ./bower_components/*, ./dist]
+
+# before_commands:
+#   - 'composer self-update'
+#   - 'composer update --prefer-stable --prefer-source --no-interaction --no-scripts --no-progress --no-suggest'
+
+tools:
+  php_code_coverage: false
+  php_code_sniffer:
+    enabled: true
+    config:
+      standard: 'PSR2'
```

Comment:
Could we use the xml file in the repo for this? Then we would have it in one place and don't need to update this file. Excludes could then also be removed from this config.